lib/input-view.js

```diff
@@ -23,7 +23,7 @@ export default class InputView extends View {
   initialize(prompt, onConfirmed) {
     this.prompt = prompt;
     this.onConfirmed = onConfirmed;
-    atom.commands.add(this.element, { 'core:confirm': this.confirm });
+    atom.commands.add(this.element, { 'core:confirm': () => this.confirm() });
```

Comment:
Does this get called without a bound `this` then?

You could also do:

```js
atom.commands.add(this.element, { 'core:confirm': this.confirm.bind(this) });
```

Depending on what `this` context you want.